andela/ah-titans-frontend
# #160510057 Update navbar

#### What does this PR do?
Implements update navbar and 404 error page

#### Description of tasks to be completed
Updates the look and feel of the navbar by utilizing a SideNav component.
Updates the 404 Page not found UI.

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans-frontend.git
```
```sh
$ cd ah-titans-frontend
```
```sh
$ git checkout -b ch-update-navbar-160510057
```
```sh
$ npm install
```
```sh
$ npm start
```
> Navigate to the homepage 

![image](https://user-images.githubusercontent.com/12577659/45824706-4b3d7e80-bcf9-11e8-9928-c8cdc394b75a.png)

> Click on the navbar burger 

![image](https://user-images.githubusercontent.com/12577659/45824739-614b3f00-bcf9-11e8-9883-b62fd314598b.png)

> Signin to see the state updated sidebar menu

![image](https://user-images.githubusercontent.com/12577659/45824837-893aa280-bcf9-11e8-9a24-4f25c1da3131.png)

#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160510057](https://www.pivotaltracker.com/story/show/160510057)
